Services/Simulations.cs

```diff
@@ -367,6 +379,82 @@ public async Task DeleteAsync(string id)
             await this.simulationsStorage.DeleteAsync(id);
         }
 
+        public async Task TrySeedAsync(string templateName)
+        {
+            try
+            {
+                // Skips if the seeding has already been complete
+                if (!await this.mainStorage.ExistsAsync(SEED_STATUS_KEY))
+                {
+                    await this.SeedSimulationsAsync(templateName);
+                    var record = new StorageRecord { Id = SEED_STATUS_KEY, Data = "Seed Completed" };
+                    await this.mainStorage.CreateAsync(record);
+                }
+                else
+                {
+                    this.log.Info("Seeding skipped.");
+                }
+            }
+            catch (Exception e)
+            {
+                var msg = "Failed to seed default simulations." + e;
+                this.log.Error(msg);
+                this.diagnosticsLogger.LogServiceError(msg);
+            }
+        }
+
+        // This creates sample simulations that will be shown on simulation dashboard by default
+        private async Task SeedSimulationsAsync(string templateName)
+        {
+            string content;
+            var fileName = templateName + ".json";
+            var filePath = Path.Combine(this.config.SeedTemplateFolder, fileName);
+            if (this.fileSystem.Exists(filePath))
+            {
+                content = this.fileSystem.ReadAllText(filePath);
+            }
+            else
+            {
+                this.log.Debug("Template not found for setting sample simulations.");
```

Comment:
What should be SEED_STATUS_KEY in this case ?

Reply:
this will continue to mark as complete if no or empty template was provided